examples/gallery/lines/hlines_vlines.py

```diff
@@ -0,0 +1,117 @@
+"""
+Horizontal and vertical lines
+=============================
+
+The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines` methods allow to plot
+horizontal and vertical lines in Cartesian, geographic and polar coordinate systems.
```

Comment:
Does it make sense to mention the Gallery example on [Line styles](https://www.pygmt.org/dev/gallery/lines/linestyles.html#sphx-glr-gallery-lines-linestyles-py) here regarding different pen settings?